source/browseMode.py

```diff
@@ -1583,6 +1585,8 @@ def script_movePastEndOfContainer(self,gesture):
 		info.expand(textInfos.UNIT_CHARACTER)
 		container=self.getEnclosingContainerRange(info)
 		if not container:
+			# Translators: Reported when the user attempts to move to the start or end of a container (list, table, etc.) 
+			# But there is no container. 

```

Comment:
Great you split this into two lines, but i'd make the split before the left paren. You can also safely make the B for the start of this line a lower case letter.